tool/gravity/cli/rpcagent.go

```diff
@@ -388,6 +443,51 @@ func rpcAgentShutdown(env *localenv.LocalEnvironment) error {
 	return trace.Wrap(err)
 }
 
+// rpcAgentStatus requests the gravity agent status from all members of the
+// cluster, then writes the information to stdout.
+// If an agent fails to return a status response, the agent will be considered
+// `Offline` and will display an empty version column.
+func rpcAgentStatus(env *localenv.LocalEnvironment) error {
+	env.PrintStep("Collecting RPC agent status")
+
+	operator, err := env.SiteOperator()
+	if err != nil {
+		return trace.Wrap(err)
+	}
+
+	creds, err := fsm.GetClientCredentials()
+	if err != nil {
+		return trace.Wrap(err)
+	}
+
+	cluster, err := operator.GetLocalSite()
+	if err != nil {
+		return trace.Wrap(err)
+	}
+
+	timeout, err := utils.GetenvDuration(constants.AgentStatusTimeoutEnvVar)
+	if err != nil {
+		timeout = defaults.AgentStatusTimeout
+	}
+
+	ctx, cancel := context.WithTimeout(context.Background(), timeout)
+	defer cancel()
+
+	statusList := rpc.CollectAgentStatus(ctx, cluster.ClusterState.Servers, fsm.NewAgentRunner(creds))
+
+	t := goterm.NewTable(0, 10, 5, ' ', 0)
+	common.PrintTableHeader(t, []string{"Hostname", "Address", "Status", "Version"})
+	for _, status := range statusList {
+		fmt.Fprintf(t, "%s\t%s\t%s\t%s\n", status.Hostname, status.Address, status.Status, status.Version)
+		if status.Error != nil {
+			log.WithError(err).Debugf("Failed to collect query agent version on %s.", status.Address)
+		}
+	}
+	fmt.Println(t.String())
+
+	return trace.Wrap(err)
```

Comment:
It's a bit weird - what is this error that's returned here? The duration parsing error from above?